superset-frontend/webpack.config.js

```diff
@@ -496,7 +496,7 @@ const config = {
     'react/lib/ReactContext': true,
   },
   plugins,
-  devtool: 'source-map',
+  devtool: isDevMode ? 'source-map' : false,
```

Comment:
read somewhere that turning off source maps in prod may help memory usage, but also thinking it's better to not have source maps in prod as it's lighter weight / more secure

Reply:
The source maps are SUPER helpful in prod, actually... it allows us to help pinpoint runtime errors in the codebase rather than see garbage minimized JS and stacktraces. I would leave these on.